test/dlrm/dlrm_kernel_base_impl.hpp

```diff
@@ -358,9 +359,9 @@ namespace rocwmma
 
             mElapsedTimeMs        = float64_t(timeMs);
             mTotalGFlops          = calculateGFlops(outputSize, mB, mK);
-            mMeasuredGFlopsPerSec = calculateGFlopsPerSec(outputSize, mB, mK, mElapsedTimeMs)
+            mMeasuredTFlopsPerSec = calculateTFlopsPerSec(outputSize, mB, mK, mElapsedTimeMs)
                                     * static_cast<float64_t>(mRepeats);
-            mEfficiency = mMeasuredGFlopsPerSec / devicePeakGFlopsPerSec * 100.0;
+            mEfficiency = (int)((double)(mMeasuredTFlopsPerSec / devicePeakGFlopsPerSec * 100000.0));
```

Comment:
Can you use round(result) from <cmath>. This will round the result and give you an integer result